nbs/41_tabular.data.ipynb

```diff
@@ -75,9 +75,9 @@
     "        return to.dataloaders(path=path, **kwargs)\n",
```

Comment:
<p>It's better to use markdown prose than comments in this situation. Remember - this becomes documentation, it's not just a test. So perhaps create a markdown cell above this that explains that this is how you can do a prediction with tabular? I'm not sure you need <em>all</em> those <code>test_ne</code>  lines BTW. Perhaps just keep one of them?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2716/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Got it. I've moved the comments to a markdown cell and cut down the redundant <code>test_ne()</code></p>